src/react-components/room/UserProfileSidebar.js

```diff
@@ -38,7 +48,25 @@ export function UserProfileSidebar({
   ...rest
 }) {
   const intl = useIntl();
-  const [minLevel, maxLevel, levelStep, level, updateGainMultiplier, isMuted, updateMuted] = useAvatarVolume(userId);
+  const sliderRef = useRef();
+  const onMultiplierChanged = useCallback(
+    multiplier => {
+      sliderRef.current.setValue(calcLevel(multiplier));
+    },
+    [sliderRef]
+  );
+  const [multiplier, prevMultiplier, updateMultiplier, isMuted, updateMuted] = useAvatarVolume(
+    userId,
+    onMultiplierChanged
+  );
+  const onLevelChanged = useCallback(
+    level => {
+      updateMultiplier(calcGainMultiplier(level));
+    },
+    [updateMultiplier]
+  );
+  const newLevel = calcLevel(multiplier);
+  const step = multiplier > prevMultiplier ? calcLevelStepDown(newLevel) : calcLevelStepUp(newLevel);
```

Comment:
Not really following why this step needs to be dynamic? Tried using the slider too and I can't really tell when its using one or the other. Is this a specific requirement from the designs or something?